api/institutions/authentication.py

```diff
@@ -388,6 +389,7 @@ def authenticate(self, request):
                 sso_mail=sso_email,
                 sso_department=department,
             )
+            user.add_system_tag(institution_source_tag(secondary_institution._id))
```

Comment:
```python
if is_created:
    user.add_system_tag(institution_source_tag(secondary_institution._id))
```

For consistency, only add system when new user is created for secondary institution